selenium_tests/management/commands/snapshot_dashboard_states.py

```diff
@@ -224,10 +225,6 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         os.environ['DJANGO_LIVE_TEST_SERVER_ADDRESS'] = '0.0.0.0:8286'
-        os.environ['ELASTICSEARCH_INDEX'] = 'testindex'
```

Comment:
None of these were actually being used but the default values for the other three were sufficient. `DJANGO_LIVE_TEST_SERVER_ADDRESS` is special since it's used explicitly as an environment variable and not as a means to set a setting property.